Makefile

```diff
@@ -596,20 +596,24 @@ image-to-cluster: image openscap-image namespace openshift-user  ## Builds and p
 	$(eval IMG = image-registry.openshift-image-registry.svc:5000/openshift/$(APP_NAME):$(TAG))
 	$(eval OPENSCAP_IMAGE = image-registry.openshift-image-registry.svc:5000/openshift/$(OPENSCAP_NAME):$(OPENSCAP_TAG))
 
-.PHONY: e2e-content-images
-e2e-content-images:  ## Build the e2e-content-image
+.PHONY: build-e2e-content-images
+build-e2e-content-images: ## Build content image for end-to-end testing.
 	RUNTIME=$(RUNTIME) images/testcontent/broken-content.sh build ${E2E_BROKEN_CONTENT_IMAGE_PATH}
 
-.PHONY: push-e2e-content
-push-e2e-content: e2e-content-images  ## Build and push the e2e-content-images
+.PHONY: push-e2e-content-images
+push-e2e-content-images: build-e2e-content-images ##Push the content image for end-to-end testing to an image repository.
 	RUNTIME=$(RUNTIME) images/testcontent/broken-content.sh push ${E2E_BROKEN_CONTENT_IMAGE_PATH}
 
-.PHONY: must-gather-image
-must-gather-image:  ## Build the must-gather image
+.PHONY: build-must-gather-image
+build-must-gather-image: ## Build the must-gather image.
 	$(RUNTIME) build -t $(MUST_GATHER_IMAGE_PATH):$(MUST_GATHER_IMAGE_TAG) -f images/must-gather/Dockerfile .
 
+.PHONY: push-must-gather-image
+push-must-gather-image: ## Push the must-gather image to an image repository (default: quay.io/compliance-operator/must-gather).
+	$(RUNTIME) push ${MUST_GATHER_IMAGE_PATH}:${MUST_GATHER_IMAGE_TAG}
+
 .PHONY: must-gather
-must-gather: must-gather-image must-gather-push  ## Build and push the must-gather image
+must-gather: must-gather-image must-gather-push ## Build the must-gather image and push it to an image repository (default: quay.io/compliance-operator/must-gather).
```

Comment:
Should the `must-gather` target depend on `build-must-gather-image` and `push-must-gather-image`?

Reply:
It looks like the `must-gather` target was designed to take care of the image prior to collecting anything.

Do you have another workflow in mind?

Reply:
Sorry, I was unclear. The Makefile line with your patch looks like this:
```
.PHONY: must-gather                                                                                                                                                                                                                                                                                                           
must-gather: must-gather-image must-gather-push ## Build the must-gather image and push it to an image repository (default: quay.io/compliance-operator/must-gather). 
```
So it depends on `must-gather-image` and `must-gather-push`. But I don't see neither of them in the Makefile. I do see that you renamed `must-gather-image` to `build-must-gather-image` in this patch and added `push-must-gather-image` do I wonder if this line should instead read:
```
.PHONY: must-gather                                                                                                                                                                                                                                                                                                           
must-gather: build-must-gather-image push-must-gather-image ## Build the must-gather image and push it to an image repository (default: quay.io/compliance-operator/must-gather). 
```